sso/admin.py

```diff
@@ -1,8 +1,20 @@
 from django.contrib import admin
 
-from sso.models import AutologinClient
+from sso.models import AutologinClient, User
 
 
 @admin.register(AutologinClient)
 class AutologinClientAdmin(admin.ModelAdmin):
     list_display = ["id", "oidc_client"]
+
+
+@admin.register(User)
+class UserAdmin(admin.ModelAdmin):
+    list_display = (
+        "username",
+        "email",
+        "first_name",
+        "last_name",
+        "is_active",
+        "is_staff",
```

Comment:
du coup, j'arrive pas trop à capter si on peut ajouter `preferred_username` et `name` ou si ce sont des claims ajoutés après ?

Reply:
Non, ils n'apparaissent pas dans le modèle.

> ERRORS:
<class 'sso.admin.UserAdmin'>: (admin.E108) The value of 'list_display[6]' refers to 'preferred_username', which is not a callable, an attribute of 'UserAdmin', or an attribute or method on 'sso.User'.
<class 'sso.admin.UserAdmin'>: (admin.E108) The value of 'list_display[7]' refers to 'name', which is not a callable, an attribute of 'UserAdmin', or an attribute or method on 'sso.User'.